source/develop/release-management/features/ovirt-ovn-provider.md

```diff
@@ -0,0 +1,356 @@
+An OVS integration bridge must be created on the host. This bridge
+is used to connect all OVN managed nics.
+Each host is identified by its "chassis id". This chassis id is used to route
+the network traffic to the appropriate host.
+
+## oVirt OVN Provider architecture
+
+The oVirt OVN provider consists of two parts:
+
+*   The OVN provider - a proxy between the oVirt engine and
+    the OVN North DB
+
+*   The OVN VIF driver - connects the oVirt vNIC to the proper OVS bridge
+    and OVN logical network
+
+### oVirt OVN Provider
+
+The oVirt OVN provider is a proxy between the oVirt engine and
+the OVN North DB. The provider implements the Openstack Rest API
+to allow it to be used by oVirt using the external network provider
+mechanism. REST queries from oVirt are translated into appropriate
+OVN database queries.
+The provider handles the following requests:
+
+*   GET Networks - retrieves a list of all Logical Switches from the OVN north DB Logical_Switch table
+*   GET Network - retrieves a specific Logical Switch from the OVN north DB Logical_Switch table
+*   POST Network - updates a specific Logical Switch from the OVN north DB Logical_Switch table
+*   DELETE Network - deletes a specific Logical Switch from the OVN north DB Logical_Switch table
+
+*   GET Ports - retrieves a list of all Logical Switch Ports from the OVN north DB Logical_Switch_Port table
+*   GET Port - retrieves a specific Logical Switch Port from the OVN north DB Logical_Switch_Port table
+*   POST Port - updates a specific Logical Switch Port from the OVN north DB Logical_Switch_Port table
+*   DELETE Port - deletes a specific Logical Switch Port from the OVN north DB Logical_Switch_Port table
+
+*   GET Subnets - not implemented (no OVN implementation yet)
+*   GET Subnet - not implemented (no OVN implementation yet)
+*   POST Subnet - not implemented (no OVN implementation yet)
+*   DELETE Subnet - not implemented (no OVN implementation yet)
+
+The oVirt OVN provider can be located anywhere in the system. The address of the
+provider is specified when the provider is added to oVirt, so the two can be on
+any two hosts if needed.
+The provider contacts OVN using a web socket, so the two can also be located on
+separate hosts.
+
+### oVirt OVN VIF Driver
+
+The oVirt OVN VIF driver connects the oVirt vNIC to the proper OVS bridge
+and OVN logical network. When an OVN port (added by the provider) is plugged
+into an OVS bridge, OVN Controller will retrieve information about the port
+and the logical network it belongs to from OVN south DB, and create the required
+open flows on the local OVS switch. The OVN controller will also update the
+binding tables on the OVN south DB, causing all other OVN controllers on other
+hosts to update their open flow tables, to allow connections to the newly added
+port (if ports belonging to this logical network are present on this OVS switch).
+The oVirt OVN VIF driver will also take care of providing the data about the
```

Comment:
Done,  but:
Just in case, I'm adding:
"The GetStatistics VDSM command needs to be checked."